moshi/src/main/java/com/squareup/moshi/JsonReader.java

```diff
@@ -528,6 +532,11 @@ private Options(String[] strings, okio.Options doubleQuoteSuffix) {
       this.doubleQuoteSuffix = doubleQuoteSuffix;
     }
 
+    /** Returns a copy of this {@link Options Option's} strings. */
+    public List<String> strings() {
+      return unmodifiableList(Arrays.asList(strings.clone()));
```

Comment:
Sorry there's a ton of churn on this line but if you're wrapping in unmodifiable you also don't need the clone 😶

Reply:
De-cloned! 31cf8d1